game/models.py

```
@@ -70,7 +70,30 @@ class Meta:
+        related_name="progress"
+    )
+
+    xp = models.PositiveIntegerField(
+        default=0
+    )
+
+    level = models.PositiveIntegerField(
+        default=1
+    )
+
+    def __str__(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)